home/tests.py

```diff
@@ -1,3 +1,40 @@
-# from django.test import TestCase
+from django.contrib.auth.models import User
+from home.models import Profile, Question, Answer
+import pytest
 
-# Create your tests here.
+
+@pytest.mark.django_db
+def test_Add_Profile():
+    user = User.objects.create_user(username="test_user", password="testtest", email="test@test.com")
+    profile = Profile(user=user)
+    profile.save()
+    assert profile.user.id >= 0  # check if user saved successfully and get an id
+    assert profile.gender == 'U'  # default gender is U
+    assert profile.is_blocked is False
+
+
+@pytest.fixture
+def profile():
+    profile = Profile.create(username="test_user", password="testtest", email="test@test.com")
+    profile.save()
+    return profile
+
+
+@pytest.mark.django_db
+def test_Add_Answer(profile):
+    question = Question.objects.first()
+    if profile and question:
+        answer = Answer(profile=profile, question=question, content='Test')
```

Comment:
What is the logic theses asserts are testing? Are they desting the default values for the attributes blow? The test name does not help in understanding that....